ERCS/erc-7798.md

```diff
@@ -0,0 +1,200 @@
+---
+eip: 7798
+title: Tap to Pay
+description: Initializing contactless payment transactions from EVM wallets
+author: Amhed Herrera (@amhed), Justin Lee (@JustinDLee), Arjun Dureja (@arjun-dureja)
+discussions-to: https://ethereum-magicians.org/t/erc-7798-contactless-payment/21501
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-25
+requires: 20, 681, 712
+---
+
+## Abstract
+This ERC defines a standard for contactless payment transactions that can allow for interoperable customer to merchant onchain payments, regardless of the wallets the customer and merchant are using.
```

Comment:
The abstract should contain enough detail to give the reader a high-level (but still technical) overview of _how_ your proposal accomplishes its goals.